lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -571,6 +570,7 @@ def _run_pdfminer(pdf_path, hash_key, use_cache):
 
         return pdfminer_layout
 
+    @requires_modules("easyocr", extra="local-inference")
```

Comment:
Instead of doing this, why don't we shift the initialization of the ocr table reader to the `extract_ocr` method? Would keep both `pytesseract` and `easyocr` together which makes more sense in my mind

Reply:
I like that idea from a clarity perspective, but it does mean that the reader would be initialized once per doc, rather than once per ArynPdfPartitioner instance. Not sure that matters much, though I guess if it did it would be for the service which uses long-lived actors. 

I think it's reasonable to experiment with, but I might suggest we push that refactoring to a different PR just to reduce complexity here.